cluster-install/neptune.yaml

```diff
@@ -1,3 +1,16 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    app.kubernetes.io/component: manager
+    app.kubernetes.io/created-by: neptune
+    app.kubernetes.io/instance: system
+    app.kubernetes.io/managed-by: kustomize
+    app.kubernetes.io/name: namespace
+    app.kubernetes.io/part-of: neptune
+    control-plane: controller-manager
+  name: astra-connector
+---
```

Comment:
Re-added this to unblock polaris qual pipeline (would've had to wait for a neptune PR merge otherwise).